src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/Startup.cs

```diff
@@ -120,7 +120,9 @@ public void ConfigureServices(IServiceCollection services)
 #else
             services.AddControllersWithViews();
 #endif
-            services.AddRazorPages();
+#if (OrganizationalAuth || IndividualAuth)
```

Comment:
Does org auth use razor pages?

Reply:
I was pretty sure it uses the identity UI. Let me check

Reply:
Confirmed Microsoft.AspNetCore.Authentication.AzureAD.UI uses Razorpages